src/views/CohortPage.js

```diff
@@ -1,78 +1,50 @@
 import React, { useEffect } from 'react';
-import { Route, useRouteMatch, useHistory, withRouter } from 'react-router-dom';
+import { useRouteMatch } from 'react-router-dom';
 import { useDispatch, useSelector } from 'react-redux';
-import { withTheme } from 'styled-components';
+import isEmpty from 'lodash/isEmpty';
 
 import { Sidebar } from '../components/Sidebar/Sidebar';
-import { BarGraph } from '../components/Graphs/BarGraph';
-import { CohortPageStyle } from './CohortPageStyle';
+import { CohortPageStyle, NotificationStyle } from './CohortPageStyle';
 import { GraphSectionStyle } from './GraphSectionStyle';
 import { fetchSingleCohortData } from '../store/actions/singleCohort.actions';
-import { orderBars, sortDesc } from '../utils/order';
+import { GraphRoutes } from './GraphRoutes';
 
-const CohortPage = ({ theme }) => {
-  const history = useHistory();
-  const cohortID = history.location.pathname.split('/')[2];
-  const { path } = useRouteMatch();
+const renderView = (isPending, isFailure, cohort, cohortID) => {
+  if (isPending || isEmpty(cohort)) {
+    return (
```

Comment:
it this necessary `isEmpty(cohort)`? @IrinaGM and I checked it and it seems like it is not.

Reply:
Removed!